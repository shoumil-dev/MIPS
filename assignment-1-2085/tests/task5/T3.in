3
90 
40 
23
1
