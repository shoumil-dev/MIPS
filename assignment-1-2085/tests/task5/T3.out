MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

Array length: Enter num: Enter num: Enter num: Enter r: 90 
40 
23 

